content/logs/aws.md

```diff
@@ -0,0 +1,133 @@
+---
+title: AWS Log Collection
+kind: Documentation
+autotocdepth: 2
+hideguides: true
+customnav: lognav
+beta: true
+---
+
+## Overview
+
+This documentation explains how to create a Lambda function in AWS to forward S3 logs to Datadog.
+
+S3 logging is used by well known services such as: ELB, S3, CloudTrail, VPC, CloudFront and CloudWatch. There are then all eligible to this method.
+Using a Lambda function is elegant as it avoids the hassle of creating processes that would pull periodically the incoming S3 logs. The Lambda function is called whenever a new S3 log file is made available.
+Features:
+
+* Use AWS Lambda to re-route triggered S3 events to Datadog
+* ELB, S3, CloudTrail, VPC and CloudFont logs can be forwarded
+* SSL Security
+* JSON events providing details about S3 documents forwarded
+* Structured meta-information can be attached to the event
+
+## Setup
+### Installation
+#### Create a new lambda function
+
+1. Navigate to the [Lambda Console](https://console.aws.amazon.com/lambda/home?region=us-east-1) and create a new function:
+    {{< img src="logs/aws/create_lambda_function.png" alt="Create Lambda function" >}}
+
+2. Select **Author from scratch** and give the function a unique name.
+    {{< img src="logs/aws/author_from_scratch.png" alt="Author from Scratch" >}}
+3. For `Role`, select **Create new role from template(s)** and give the role a unique name
+4. Under Policy templates, search for and select **s3 object read-only permissions**
+    {{< img src="logs/aws/basic_information.png" alt="Basic Information" >}}
+5. Select **Create Function**
+
+#### Provide the code and configure the lambda
+
+6. Copy and paste the code from [This repo](https://github.com/DataDog/dd-aws-lambda-functions/blob/master/Log/lambda_function.py) into the function code area. (Can you do this for python still??)
```

Comment:
`(Can you do this for python still??)` to be removed

Reply:
👍